tests/integration/kubernetes/runtimeclass_workloads/k8s-policy-rc.yaml

```diff
@@ -27,6 +27,7 @@ spec:
         volumeMounts:
           - name: host-empty-vol
             mountPath: "/host/cache"
+            readOnly: true
```

Comment:
Should we rather add a new read-only emptyDir mount to ensure both paths are tested?